simple-facebook-instant-articles.php

```diff
@@ -73,7 +72,7 @@ public function __construct( $file, $version ) {
 		$this->file          = $file;
 		$this->template_path = trailingslashit( $this->dir ) . 'templates/';
 		$this->home_url      = trailingslashit( home_url() );
-		$this->endpoint      = apply_filters( 'simple_fb_article_endpoint', 'fb-instant' );
```

Comment:
This should prolly be 
`$this->endpoint      = apply_filters( 'simple_fb_article_endpoint', $this->token );` and then have `$token` set to an empty string.